.pipelines/cni/k8s-e2e/k8s-e2e-step-template.yaml

```diff
@@ -0,0 +1,57 @@
+parameters:
+  testName: ""
+  name: ""
+  clusterName: ""
+  ginkgoFocus: ""
+  ginkgoSkip: ""
+  os: ""
+  processes: "" # Number of parallel processes
+  attempts: ""
+
+
+steps:
+  - script: |
+      set -ex
+
+      # ginkgoSkip cant handle only |LinuxOnly. Need to have check
+      if ${{ lower(and(ge(length(parameters.ginkgoSkip), 1), eq(parameters.os, 'windows'))) }}
+      then
+        SKIP="|LinuxOnly"
+      elif ${{ lower(eq(parameters.os, 'windows')) }}
```

Comment:
`lower` converts the string to lower case, why do we have it outside the `eq` ?

Reply:
Converting it to lowercase is a bash if () constraint. It cannot interpret "False" as intended, as "false" is the bash command for a false boolean. Likewise, with "true" versus "True".